### PR TITLE
More bugzilla-reporter fixes

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by michalfabik** [comment]
Makes sense, thanks.

File: src/plugins/python/reporter_bugzilla.py [hunk 1]
```diff
@@ -117,6 +117,8 @@ def set_settings(bz: Dict, settings: Dict):
         environ = settings.get('DontMatchComponents')
     bz['b_DontMatchComponents'] = environ or ''
 
+    if bz.get('b_create_private'):
+        bz['b_create_private'] = False
```
Comment (mgrabovsky) [inline src/plugins/python/reporter_bugzilla.py:121]: “If `b_create_private` is already set, override it to `False`”? Is this intended?

Comment (msrb) [reply src/plugins/python/reporter_bugzilla.py:121]: Ouch, no, of course not. I am missing "not" there. Good catch!

Comment (msrb) [reply src/plugins/python/reporter_bugzilla.py:121]: https://github.com/abrt/libreport/pull/760

**Pull request merged**